### PR TITLE
[ops] seed/wipe cli tool

### PR DESCRIPTION
**Pull request by dZefa**
- adds user(10) and room(100) seed data
- adds commands to seed or wipe data
  - two commands available are: "yarn db seed" and "yarn db wipe"
  - seed command wipes appropriate collection if said collection exists, before seeding data

### CODE REVIEW

File: bin/seedDB/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import path from 'path';
+require('dotenv').config({ path: path.resolve(__dirname, '../../rest-server/.env') });
+
+import dbConfigs from './config';
```
Comment (obber) [inline bin/seedDB/index.js]: I was running into a bug where it was loading the .env settings AFTER the config file was loaded. This was an issue because `exec()` was being run and `process.env.MONGO_DB_NAME` was undefined.

In my `bin/seedDB/config.js` file, I put this towards the top:

```javascript
console.log('process.env.MONGO_DB_NAME = ', process.env.MONGO_DB_NAME);
setTimeout(() => {
  console.log('process.env.MONGO_DB_NAME = ', process.env.MONGO_DB_NAME);
}, 0);
```

And it produced these console logs:

```
➜  codesling git:(dZefa-feat/seedcli) ✗ yarn db seed
yarn db v0.17.10
$ node bin/entry.js seedDB seed
process.env.MONGO_DB_NAME =  undefined
successfully fulfilled seed command.
process.env.MONGO_DB_NAME =  codesling
```

Low and behold, this is caused because the `import` statements are executed in a depth-first (recursive) fashion, which means every imported module has to know what other files it imports. Meaning, before code is run, the import dependency tree is built, and the deepest dependency executes first. 

TLDR; It's as if ES6 `imports` gets hoisted to the top :(

Source: https://hacks.mozilla.org/2015/08/es6-in-depth-modules/ (Read above the Export List section)

File: bin/seedDB/config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import path from 'path';
+import { exec } from 'child_process';
+
+const userSeedCmd = `mongoimport --db ${process.env.MONGO_DB_NAME} --collection users --drop --file ${path.resolve(__dirname, './usersSeedData.json')} --jsonArray`;
```
Comment (obber) [inline bin/seedDB/config.js]: We likely can't define the shell command to execute in the parent scope of this file. 

It's gotta be inside of the function that executes (inside the arrow function on line 9), since the string literal is defined at runtime, and when this file runs, it is being run before `.env` is loaded (see my long comment below about imports being hoisted)

File: package.json [hunk 3]
```diff
@@ -32,6 +33,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.0",
     "babel-register": "^6.26.0",
+    "dotenv": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (obber) [reply package.json]: Oh, and if you don't have yarn installed, you'll need to install it globally using `npm install -g yarn`

File: bin/seedDB/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+if (!process.env.MONGO_DB_NAME) {
+  console.log(`env file does not exist. \n Please make sure env setup is complete.`);
+  process.exit(1);
+}
+
+try {
+  Object
+    .keys(commandObj)
```
[the remark this answers is not in the document]
Comment (obber) [reply bin/seedDB/index.js]: Speaking of which, we should probably change it here:

https://github.com/codesling/codesling.io/blob/master/bin/buildEnv/createFile.js

Feel free to add this change to this PR :)

File: bin/seedDB/config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dZefa) [reply bin/seedDB/config.js]: I see. Then should I take this one step further and refactor the shell commands to their own file that contains only shell commands?

[the remark this answers is not in the document]
Comment (obber) [reply bin/seedDB/config.js]: 👍

File: bin/seedDB/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (dZefa) [reply bin/seedDB/index.js]: @obber did you mean in https://github.com/codesling/codesling.io/blob/master/bin/buildEnv/help.js ? seems like lodash was being used in createFile

[the remark this answers is not in the document]
Comment (obber) [reply bin/seedDB/index.js]: Actually don't worry bout this for this PR! We'll get it another time :)

**Comment by obber** [comment]
Also, because of the docs I PRed and you merged in earlier, it looks like there is a conflict in `docs/README.md` now.

Feel free to rebase and push to a new branch, or rebase and force push to this current branch. You may need to close out this PR and open a new one in the process :)
